### PR TITLE
Initialize adding the Simplified Chinese (zh-CN) translation of the fpm documents

### PR DESCRIPTION
**Pull request by zoziha**
- [x] add the Chinese translation.

CN Pages: https://zoziha.github.io/fpm-docs/cn/ (outdated)

### CODE REVIEW

**Comment by awvwgk** [comment]
I think `cn` is not matching any language sphinx-intl is aware of, however it has two entries for Chinese:

- `zh_CN` – Simplified Chinese
- `zh_TW` – Traditional Chinese

Using either of those would also use translations for the blog and table of contents. If you prefer `cn` as path for the URL I can have a look how we can keep this path and still get the right translation from the sphinx-intl for the theme.

**Comment by zoziha** [comment]
Thanks, I'll replace `cn` with `zh_CN`.

File: pages/conf.py [hunk 1]
```diff
@@ -24,18 +24,20 @@
 
 _extra_navbar = """
 <div class="sd-fs-6">
-<a  href="../en" target="_blank">en</a>
+<a href="../en">en</a>
 ·
-<a  href="../de" target="_blank">de</a>
+<a href="../de">de</a>
+·
+<a href="../zh_CN">zh_CN</a>
```
Comment (awvwgk) [inline pages/conf.py:31]: Is `cn`, `zh` or the full `zh_CN` more intuitive here as language identifier?
```suggestion
<a href="../zh_CN">cn</a>
```

Comment (zoziha) [reply pages/conf.py:31]: `cn` seems fine, `zh_CN` name is indeed too long and eye-catching.

**Review by awvwgk — APPROVED** [review]
Thanks a lot, I'll go ahead and merge.

**Pull request merged**